src/util.js

```diff
@@ -198,6 +198,47 @@ exports.rateLimited = function(intervalMillis, func) {
   return schedule;
 };
 
+/**
+ * Checks if elem is 'open', i.e. would display in the browser
+ */
+exports.isOpen = function(elem) {
+  if (elem.textContent.length === 0) {
```

Comment:
perhaps clearer:

// simple case
if (elem.textContent.length > 0) {
  return true;
}

// check for BR that holds it open:
....